platform/android/app/src/main/kotlin/coop/polypoly/polypod/polyOut/PolyOut.kt

```diff
@@ -7,22 +7,19 @@ import android.provider.OpenableColumns
 import coop.polypoly.polypod.Preferences
 import coop.polypoly.polypod.features.FeatureStorage
 import coop.polypoly.polypod.polyNav.ZipTools
-import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.async
 import kotlinx.coroutines.supervisorScope
 import java.io.File
 import java.nio.ByteBuffer
 import java.util.UUID
-import kotlin.coroutines.EmptyCoroutineContext
 
 open class PolyOut(
     val context: Context
 ) {
     private var readDirCache =
         mutableMapOf<String, Array<Map<String, String>>>()
     private var statCache = mutableMapOf<String, MutableMap<String, String>>()
-    private val coroutineScope = CoroutineScope(EmptyCoroutineContext)
```

Comment:
This is probably not used, right?

Reply:
yep :) found it on the go

Reply:
Was there a warning somewhere? Are we prepared to detect these things either on the IDE or during compilation?

Reply:
nope there was nothing. I was just looking on uses of coroutines in the code and bumped into this but it's unused :(

Reply:
So our linting is not all it's cracked up to be... :cry: